source/dofs/dof_handler_policy.cc

```diff
@@ -186,13 +190,15 @@ namespace internal
 
               // double check whether the newly created entries make
               // any sense at all
-              for (unsigned int i = 0; i < identities->size(); ++i)
+              for (const auto &identity : *identities)
                 {
-                  Assert((*identities)[i].first <
-                           fe1.template n_dofs_per_object<structdim>(face_no),
+                  Assert(identity.first <
```

Comment:
To get rid of the CI complaints:
```suggestion
                  (void)identity;
                  Assert(identity.first <
```